internals/crypto/ciphertext.go

```diff
@@ -104,9 +104,11 @@ func newEncodedCiphertextMetadata(metadataList map[string]string) encodedCiphert
 	metadata := ""
 
 	// Sort all the keys of the metadataList so that metadata is always in alphabetical order.
-	var keys []string
+	keys := make([]string, len(metadataList))
+	i := 0
 	for k := range metadataList {
```

Comment:
The parameter `metadataList` is confusingly named. Because it's a `list` I exptected it to be a slice and expected to see simpler iteration logic. Upon inspection I realized it's a map, hence the conversion. Not blocking at all, but would be nice to name it intuitively.